jibri/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM jitsi/base-java
+
+ARG JITSI_RELEASE=unstable
```

Comment:
Let's drop this and just use unstable for this directly, since it's "temporary" until we move jibri to stable.

Reply:
I guess need a comment in README about it, because it's it is not clear.

Reply:
I don't think it's worth mentioning. We should mention that recording is "beta" though.